datatypes/src/operations/spatial_relation.rs

```diff
@@ -3,8 +3,3 @@
 pub trait Contains<T> {
     fn contains(&self, other: &T) -> bool;
 }
-
-/// This method returns `true` iff does this `Self` intersect with the input `T`
```

Comment:
why is this no longer needed?

Reply:
it is never used

Reply:
[6849724](https://github.com/geo-engine/geoengine/pull/973/commits/684972467e851106b49dca727adc36c1d5595550)

Reply:
?